h5py/_hl/group.py

```diff
@@ -267,16 +263,15 @@ def create_dataset_like(self, name, other, **kwupdate):
 
         return self.create_dataset(name, **kwupdate)
 
-    def require_group(self, name):
```

Comment:
Should we be checking what `track_order` is, and raising if the group doesn't match the properties?

Reply:
Sorry, I don't get it,  in line #274, it will call create_group, then it will check `track_order` in it.

Reply:
This is similar to https://github.com/h5py/h5py/pull/897, where we don't know if the arguments are trying to check whether the group is being tracked or not (e.g. `g.require_group("test", True)` could mean "create this group if it doesn't exist, and start tracking changes" or "create this group and track changes, if the group doesn't exist"). We need to know how the user wants tracking to be treated (and if we should error).

Reply:
I think I understand your word now.
but sadly, I can't do it like in reuqire_dataset:
https://github.com/h5py/h5py/blob/fba98fd5b1b0ed39074b292323980096a5637448/h5py/_hl/group.py#L224-L234
cause `group` have no property like dataset. so I can't check it like: `if not track_order == grp.track_order`.

Actually, I want to check `original group` and `require group` in testcase before, but I realized it didn't work.